week_1/Project/youtube-fe/styles/video.css

```diff
@@ -0,0 +1,56 @@
+.video-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    column-gap: 16px;
+    row-gap: 40px;
+}
+
+.thumbnail-row {
+    margin-bottom: 12px;
+    position: relative;
+}
+
+.thumbnail {
+    width: 100%;
+    border-radius: 15px;
+}
+
+.video-time {
+    background-color: black;
+    color: white;
+    position: absolute;
```

Comment:
@codal-kcolaco what is `position: absolute;` here? and what values can he here in position? and what all that is used  for

Reply:
`position: absolute` is used to take an element out of normal html document flow. The element with this attribute won't affection the position of other elements on the page. Here I have used it to show a length of a youtube video. When we scroll down it will remain in its space relative to the thumbnail. Values like `top`, `bottom`, `right ` and  `left` are used to position the element where we want.